cstring.h

```diff
@@ -363,6 +363,40 @@ int64_t stringFindFirstCharArr(TString s, const char *pattern) {
     return -1; 
 }
 
+double stringToDouble(TString s) {
+    double result = 0.0;
+    int sign = 1;
+    int decimalIndex = -1;
+    if (s.size > 0 && s.data[0] == '-') {
+        sign = -1;
+        stringRemove(&s, 0, 1);
```

Comment:
Deleting the first element takes O(n), it's too much
It'll be better just set some sort of start_position to 1 and then start loop not from 0 but from start_position